source/fundamentals/connection/mongoclientsettings.txt

```diff
@@ -218,23 +218,30 @@ settings to modify the driver's behavior:
 
    * - ``srvHost()``
      - Sets the host name to use to look up an SRV DNS record to find the MongoDB hosts.
-     
+
        .. note::
-        
+
           When setting ``srvHost``, the driver does not process any
-          associated TXT records associated with the host. 
+          associated TXT records associated with the host.
 
           If you want to enable the processing of TXT records, you must
-          specify the SRV host in the connection string using the 
+          specify the SRV host in the connection string using the
           ``applyConnectionString()`` method.
 
           .. code-block:: java
              :emphasize-lines: 3
-        
+
              MongoClient mongoClient = MongoClients.create(
                 MongoClientSettings.builder()
                    .applyConnectionString(new ConnectionString("mongodb+srv://host1.acme.com")))
 
+   * - ``srvMaxHosts()``
+     - | Sets the maximum number of hosts the driver can connect to when using 
+         the DNS seedlist (SRV) connection protocol, identified by the 
+         ``mongodb+srv`` connection string prefix. 
```

Comment:
```suggestion
         the DNS seedlist (SRV) connection protocol.
```

Nit/Suggestion: Showing what the DNS seedlist (SRV) connection protocol is makes this description a bit lengthy. Perhaps linking to the section in the connection guide would be better? That way, there's also more information once they get there.

Reply:
I considered that, but thought there wasn't enough focused material to help the reader [there](https://docs.mongodb.com/drivers/java/sync/current/fundamentals/connection/#connection-uri). I think either way would work, but this way helps the reader identify whether they're using it immediately which can help them avoid a potential exception being thrown if they're not using SRV. Might be something to address in the Connection URI page in the future.